### ISSUE TITLE
Run the test script

### ISSUE THREAD

**Issue by pbelmann** [CampagneLaboratory/NextflowWorkbench#3]
Hi,

I tried to run the example from the tutorial but I can not start it.

You see here the configuration:
![nextflow](https://cloud.githubusercontent.com/assets/8780236/9492608/4aa9024c-4bf9-11e5-8053-63ae7b8fb244.png)

The MPS IDE reports the following nullpointer exception:

``` shell
[1918549]  ERROR - llij.ide.plugins.PluginManager - null 
java.lang.NullPointerException
    at org.campagnelab.workflow.run.plugin.NextFlow_RunParameters_Configuration$1.run(Unknown Source)
    at jetbrains.mps.ide.smodel.WorkbenchModelAccess$2.run(WorkbenchModelAccess.java:118)
    at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:896)
    at jetbrains.mps.ide.smodel.WorkbenchModelAccess.runReadAction(WorkbenchModelAccess.java:113)
    at jetbrains.mps.smodel.ModelAccessBase.runReadAction(ModelAccessBase.java:52)
    at org.campagnelab.workflow.run.plugin.NextFlow_RunParameters_Configuration.getAdditionalLibPath(Unknown Source)
    at org.campagnelab.workflow.run.plugin.NextFlow_Configuration_RunProfileState.execute(Unknown Source)
    at com.intellij.execution.runners.DefaultProgramRunner.doExecute(DefaultProgramRunner.java:33)
    at com.intellij.execution.runners.GenericProgramRunner$1.execute(GenericProgramRunner.java:43)
    at com.intellij.execution.RunProfileStarter.execute(RunProfileStarter.java:39)
    at com.intellij.execution.impl.ExecutionManagerImpl$2.run(ExecutionManagerImpl.java:203)
    at com.intellij.openapi.project.DumbServiceImpl.runWhenSmart(DumbServiceImpl.java:125)
    at com.intellij.execution.impl.ExecutionManagerImpl$1$1.run(ExecutionManagerImpl.java:168)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
    at java.awt.EventQueue.access$500(EventQueue.java:97)
    at java.awt.EventQueue$3.run(EventQueue.java:709)
    at java.awt.EventQueue$3.run(EventQueue.java:703)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
    at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:748)
    at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:577)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:384)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)

```

**Comment by fac2003** [comment]
Could you provide some information about the system where you see this exception:
- name and version of the OS,
- version and build number of the NextflowWorkbench plugin installed in MPS? 
  Thanks.

**Comment by fac2003** [comment]
Also could you try these commands and let us know the result:
ls -l ~/Library/Application\ Support/MPS32/NextflowWorkbench/

ls -l ~/.MPS32/NextflowWorkbench/

**Comment by pbelmann** [comment]
> Could you provide some information about the system where you see this exception:

sure

> ```
>  * name and version of the OS,
> ```

``` shell
>lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 14.04.3 LTS
Release:        14.04
Codename:       trusty
```

> ```
>  * version and build number of the NextflowWorkbench plugin installed in MPS? Thanks.
> ```

Reference build: 266, commit: a65d2392b1c46ed37b49f7cfc371c2a96d6abf22, branch: origin/1.0.1

> Also could you try these commands and let us know the result:
> ls -l ~/Library/Application\ Support/MPS32/NextflowWorkbench/

There is no `NextflowWorkbench` directory inside the MPS (~/projects/MPS/MPS 3.2) directory:

``` shell
> ls -l
total 5088
-rw-r--r--  1 belmann belmann    2747 Jun 12 10:15 about.txt
drwxr-xr-x  2 belmann belmann    4096 Jun 12 10:15 bin/
-rw-r--r--  1 belmann belmann     208 Jun 12 10:15 build.number
-rw-r--r--  1 belmann belmann     250 Jun 12 10:15 build.properties
drwxrwxr-x  2 belmann belmann    4096 Aug 24 12:50 help/
drwxr-xr-x  9 belmann belmann    4096 Jun 12 10:14 languages/
drwxr-xr-x  3 belmann belmann    4096 Aug 24 12:50 lib/
drwxr-xr-x  2 belmann belmann    4096 Aug 24 12:50 license/
-rwxr-xr-x  1 belmann belmann    2518 Jun 12 10:15 mps.sh*
drwxr-xr-x  2 belmann belmann    4096 Aug 24 12:50 plugin/
drwxr-xr-x 21 belmann belmann    4096 Aug 24 12:50 plugins/
-rw-r--r--  1 belmann belmann    3205 Jun 12 10:15 readme.txt
-rw-r--r--  1 belmann belmann     312 Jun 12 10:15 releaseNotes.txt
-rw-r--r--  1 belmann belmann 5154847 Jun 12 10:15 samples.zip
```

> ls -l ~/.MPS32/NextflowWorkbench/

There is no NextflowWorkbench directory in `~/.MPS32` on my system:

``` shell
> ls -l ~/.MPS32
total 8
drwxrwxr-x 11 belmann belmann 4096 Aug 26 11:08 config/
drwxrwxr-x 15 belmann belmann 4096 Aug 24 14:14 system/
```

**Comment by fac2003** [comment]
You could try this as a possible path, but if it does not work, we will need to investigate with Ubuntu and get back to you:

mkdir -p  ~/Library/Application\ Support/MPS32/
ln -s ~/.MPS32/config/plugins/NextflowWorkbench ~/Library/Application\ Support/MPS32/NextflowWorkbench

**Comment by fac2003** [comment]
We assembled a version that can help diagnose and hopefully fix this problem. You can get the plugin [here](http://campagnelab.org/?wpdmact=process&did=MTM5LmhvdGxpbms). According to our tests, on Linux, you may need to edit the idea.properties file located under bin, inside the downloaded MPS distribution, and uncomment the line:

idea.plugins.path=${user.home}/.MPS32/config/plugins

After installing the new plugin, you should be able to run, or at least more useful feedback about variables that need to be set.

**Comment by fac2003** [comment]
This issue should be fixed in 1.0.2. Please reopen if you still have the problem with this [new release](http://campagnelab.org/software/mps-plugins/).
